darwin/torch/dataset.py

```diff
@@ -343,3 +325,86 @@ def measure_weights(self, **kwargs):
             target = self.get_target(i)
             labels.extend([a["category_id"] for a in target["annotations"]])
         return self._compute_weights(labels)
+
+
+class BoundingBoxDetectionDataset(LocalDataset):
+    def __init__(self, transform: Optional[List] = None, **kwargs):
+        """See `LocalDataset` class for documentation"""
+        super().__init__(annotation_type="bounding_box", **kwargs)
+
+        self.transform = transform
+        if self.transform is not None and isinstance(self.transform, list):
+            self.transform = Compose(self.transform)
+
+    def __getitem__(self, index: int):
+        """
+        Notes
+        -----
+        The return value is a dict with the following fields:
+            image_id : int
+                Index of the image inside the dataset
+            image_path: str
+                The path to the image on the file system
+            labels : tensor(n)
+                The class label of each one of the instances
+            boxes : tensor(n, 4)
+                Coordinates of the bounding box enclosing the instances as [x, y, x, y]
+            area : float
+                Area in pixels of each one of the instances
+        """
+        img = self.get_image(index)
```

Comment:
What is the type of `img`?

Reply:
we don't have types in place in this part of the code. Would be better to make a separate PR to fix it